runtime/vm/jvminit.c

```diff
@@ -6349,6 +6349,40 @@ protectedInitializeJavaVM(J9PortLibrary* portLibrary, void * userData)
 	Xj9BreakPoint("jvminit");
 #endif
 
+        {
+		IDATA enabled = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXPRINTFLAGSFINALENABLE, NULL);
+		IDATA disabled = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXPRINTFLAGSFINALDISABLE, NULL);
+		if (enabled > disabled) {
+			UDATA maxHeapSize, maxDirectMemorySize;
+			char *howset;
+
+			/*
+			 * Emulate Hotspot -XX:+PrintFlagsFinal output for two specific flags:
+			 *
+			 * [Global flags]
+			 *    size_t MaxHeapSize                              = 4294967296                                {product} {ergonomic}
+			 *  uint64_t MaxDirectMemorySize                      = 3758096384                                {product} {default}
+			 */
+
+			j9tty_printf(PORTLIB, "[Global flags]\n");
+
+			maxHeapSize = (UDATA) vm->memoryManagerFunctions->j9gc_get_maximum_heap_size(vm);
+			howset="ergonomic";
```

Comment:
Nitpick I'd ignore if there weren't other changes:
```suggestion
			howset = "ergonomic";
```

Reply:
Applies to other sets of `howset` as well